tests/Set-DbaExtendedProtection.Tests.ps1

```diff
@@ -16,34 +16,57 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
 Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     Context "Command actually works" {
         It "Default set and returns '0 - Off'" {
-            $results = Set-DbaExtendedProtection -SqlInstance $script:instance1 -EnableException
+            $results = Set-DbaExtendedProtection -SqlInstance $script:instance1 -EnableException *>$null
             $results.ExtendedProtection -eq "0 - Off"
         }
+    }
+    Context "Command works when passed different values" {
+        BeforeAll {
+            Mock Test-ShouldProcess { $false } -ModuleName dbatools
+            Mock Invoke-ManagedComputerCommand -MockWith {
```

Comment:
Is this no longer an integration test now that we are mocking the main call by the command and manipulating what the output will look like for users that want to run our test framework locally?